src/lib/analytics.ts

```diff
@@ -0,0 +1,9 @@
+import { sendGAEvent } from "@next/third-parties/google";
+
+// ~~~~~~~~~~~~~~ CONSTANTS ~~~~~~~~~~~~~~ //
+const EVENT_LAUNCH_APP = "EVENT_LAUNCH_APP";
```

Comment:
this is set correctly?

Reply:
yeah it can be any string and it will show up in the dashboard as an event. will stick to the convention `EVENT-[event-name]` 👍